src/content/platform-integration/web/wasm.md

```diff
@@ -79,123 +37,93 @@ that has been migrated to be
 JavaScript. If WasmGC support is not detected at runtime, the JavaScript output
 is used so the application will continue to work across
-browsers.
-:::
 
-### Serve the built output with an HTTP server
-
-Flutter web WebAssembly uses multiple threads to render your application
-faster, with less jank. To do this, advanced browser features are used that
-require specific HTTP response headers.
-
-:::warning
-Flutter web applications won't run with WebAssembly unless the server is
-configured to send specific HTTP headers.
-:::
-
-| Name | Value |
-|-|-|
-| `Cross-Origin-Embedder-Policy` | `credentialless` <br> or <br> `require-corp` |
-| `Cross-Origin-Opener-Policy` | `same-origin` |
-
-{:.table}
-
-To learn more about these headers, check out
-[Load cross-origin resources without CORP headers using COEP: credentialless][coep].
-
-:::tip Serving locally with `dhttpd`
-To view the built output locally, you can use the
-[`dhttpd` package]({{site.pub-pkg}}/dhttpd).
-
-After installation, run it from the 'build/web' directory, and use the following
-command to add the necessary headers:  
-`dhttpd '--headers=Cross-Origin-Embedder-Policy=credentialless;Cross-Origin-Opener-Policy=same-origin'`
-:::
-
-[coep]: https://developer.chrome.com/blog/coep-credentialless-origin-trial
-
-### Load it in a browser
-
-As of {{last-update}},
-[only **Chromium-based browsers**](#chrome-119-or-later)
-(Version 119 or later) are able to run Flutter/Wasm content.
-
-You can verify whether the app is actually running with Wasm in two ways:
+You can verify whether the app is actually running with Wasm by checking for 
+the `dart2wasm` environment variable, set during compilation (preferred).
 
 ```dart
-// Using the environment variable set by compilation (preferred)
 const isRunningWithWasm = bool.fromEnvironment('dart.tool.dart2wasm');
-
-// Using differences in number representation to test whether native (Wasm)
-// number representation is used
-final isRunningWithWasm = identical(double.nan, double.nan);
 ```
 
-## Known limitations
+Alternatively, you can use differences in number representations
+to test whether the native (Wasm) number representaton is used.
 
-Wasm support currently has some limitations.
-The following list covers some common issues.
+```dart
+final isRunningWithWasm = identical(double.nan, double.nan);
+```
 
-### Chrome 119 or later
+## Learn more about browser compatibility
+To run a Flutter app that has been compiled to Wasm,
+you need a browser that supports [WasmGC][].
 
-As mentioned in [Load it in a browser](#load-it-in-a-browser),
-to run Flutter web apps compiled to Wasm,
-use _Chrome 119 or later_.
+[Chromium and V8][] released stable support for WasmGC in Chromium 119.
+Note that Chrome on iOS uses WebKit, which doesn't yet [support WasmGC][].
+Firefox announced stable support for WasmGC in Firefox 120,
+but currently doesn't work due to a [known limitation](#known-limitations).
```

Comment:
Not clear what the limitation is.

Reply:
Oops, this is a broken link. This is referring to the "Why not Firefox?" section, which I moved directly underneath this paragraph, so I'll remove this link and try to make that more clear.